CONTRIBUTING.md

```diff
@@ -209,12 +212,6 @@ We use the [Visual Studio Code](https://code.visualstudio.com/) editor.
 - Make sure to install [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extensions.
 - The [vscode-jest-runner](https://marketplace.visualstudio.com/items?itemName=firsttris.vscode-jest-runner) extension is also very helpful for debugging tests.
 
-## Manually Testing with Vega-Editor
-
-To manually test your changes locally, you should have a local instance of [Vega Editor](https://github.com/vega/editor) and link Vega-Lite to the editor (See [Vega Editor's README](https://github.com/vega/editor#local-testing--debugging) for instructions).
-
-To update the Vega-Lite code in the editor, you need to compile TypeScript to JavaScript. The easiest way is to run `yarn watch` in the Vega-Lite directory. This command will automatically recompile the code whenever you make changes.
```

Comment:
I didn't copy the "watch" part over since we shouldn't need it if we no longer build. But @domoritz feel free to further correct because I don't know what additional change you've made recently.